build_tools/ci/cpu_comparison/run.py

```diff
@@ -758,6 +758,30 @@ def run(self, config):
                 lower_to_aie_pipeline="objectFifo",
             )
 
+        # Test(s) of the form matmul(A,B) + truncf(C) where A:MxK, B:KxN and C:MxN
+        template_name = test_files_dir / "matmul_truncf_32x32x32_bf16_bf16.mlir"
+        identity_mat = np.eye(32, dtype=np.float32)
+        ones = np.ones(32 * 32, dtype=np.float32).reshape([32, 32])
+        lhs = ones * 192
+        input_args = generate_inputs(
+            template_name, output_dir, 1, {1: lhs, 2: identity_mat}
```

Comment:
Could we do `rhs == identity_matrix * 2` for example to make the expected output different from the input? We should also think whether these inputs are not too straightforward, resulting in only testing a very specific case. With `rhs == identity_matrix * 2` it looks reasonable though as an initial test.

Reply:
Sure. I've added the change.